src/olympia/stats/management/commands/theme_update_counts_from_file.py

```diff
@@ -91,6 +91,7 @@ def handle(self, *args, **options):
         ThemeUpdateCount.objects.filter(date=day).delete()
 
         theme_update_counts = {}
+        new_st_update_counts = {}
```

Comment:
what does `st` stand for? static theme? I'd prefer this being written out I think.

Reply:
yeah.  Better variable name suggestions that aren't stupidly long welcome.

Reply:
`new_static_theme_update_counts` is too long? :see_no_evil:

Reply:
If only we had longer lines 😆

Reply:
"st" become "stheme" for a little better readability (without having to overflow every statement onto multiple lines).